model/Core/Classes/NamespaceMap.md

```diff
@@ -0,0 +1,66 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NamespaceMap
+
+## Summary
+
+A mapping between prefixes and namespace URIs.
+
+## Description
+
+A namespace map allows the creator of a collection of Elements to suggest 
+a set of shorter identifiers ("prefixes") for particular namespace portions 
+of ElementIDs to be used in SPDX content serialization in order to provide a more
+human-readable and smaller serialized representation of the Elements.
+
+The content of a namespace map are suggestions for consistency in serialization but are not normatively enforced.
+Any serialization of the collection content SHOULD (rather than MUST) utilize the namespace map content.
+A serialization MAY choose to use the namespace map content.
+A serialization MAY choose to use prefixes and namespaces other than the namespace map content.
+A serialization MAY choose to use no prefixes at all and rather use the more verbose full ElementID IRIs.
+
+If utilized the namespace map set of prefixes and namespaces MUST be implemented in a given serialization 
```

Comment:
It took me a few passes to parse this statement.  At first, it seemed to contradict the "MAY" choose to use, but now I understand that this is describing that we must not replace the native serialization.

Suggest restructuring the sentence to start with the serialization form - something like "The prefix / namespace mapping of the serialization format must always be used to convey the namespace mapping for that serialization ..."

Reply:
I am not sure if your characterization is aligned to intent here.
Lines 22-26 all go together.
They are saying that if you are going to actual use the namespace map content as prefixes in a given serialization then how those prefixes are represented in a given serialization form (e.g., json-ld, xml, etc) varies form to form and the implementation of how they are represented in any given form is specified in the binding rules (that bind the model to a given serialization form) for that serialization form. In other words, if you are going to implement them in json-ld you must use the specified form for prefixes in the json-ld binding rules specification. For serialization forms that natively support prefixes it is a little more obvious. For serialization forms that don't and we have to define custom representations this is much more of an important clause.

I believe the whole first sentence (lines 22-24) clearly states this point as is. 
Does the above explanation clarify or do you still find the sentence unclear?